project/package.json

```diff
@@ -7,21 +7,33 @@
     "dev": "vite",
     "build": "vite build",
     "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "test": "vitest",
     "preview": "vite preview"
   },
   "dependencies": {
+    "@testing-library/react": "^14.1.0",
     "axios": "^1.6.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-test-renderer": "^18.2.0",
+    "vitest": "^0.34.6"
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^6.1.4",
     "@types/react": "^18.2.15",
     "@types/react-dom": "^18.2.7",
     "@vitejs/plugin-react": "^4.0.3",
+    "axios-mock-adapter": "^1.22.0",
     "eslint": "^8.45.0",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.3",
-    "vite": "^4.4.5"
+    "jsdom": "^22.1.0",
+    "vite": "^4.5.0"
+  },
+  "jest": {
+    "setupFilesAfterEnv": [
+      "C:/Users/saura/Desktop/master_courses/Softwere Engineering/project/space-explorer/project/src/setupTests.js"
```

Comment:
This path seems to be specific to your computer. Can you make it relative to the project so that it works for everyone of us?

Reply:
Changed the path to relative path